cloud-init.sh

```diff
@@ -9,6 +9,15 @@ aws_get_parameter() {
         --query Parameter.Value 2>/dev/null
 }
 
+# Install Cloudwatch Agent
+curl -sL https://s3.amazonaws.com/amazoncloudwatch-agent/ubuntu/amd64/latest/amazon-cloudwatch-agent.deb \
+  -o amazon-cloudwatch-agent.deb
+dpkg -i amazon-cloudwatch-agent.deb
+
+# Pull config and start Cloudwatch Agent
+amazon-cloudwatch-agent-ctl -a fetch-config -m ec2 -c ssm:${CLOUDWATCH_SYSTEM_CONFIG} -s
+amazon-cloudwatch-agent-ctl -a append-config -m ec2 -c ssm:${CLOUDWATCH_KONG_CONFIG} -s
```

Comment:
Would it make sense to keep the config files local to kong distribution/package here. I feel like storing it in SSM creates extra dependency and complexity - do we plan to share it with some other (non-kong) packages? unlikely.

Reply:
If these configs are not sensitive - yes please, let's keep them in the repo.

Reply:
(note this repo is public)

Reply:
or perhaps let's pass the configs from the variables to this module?

Reply:
OK, actually it's fine to use SSM (after looking on another PR)